tracing-core/src/field.rs

```diff
@@ -70,6 +70,7 @@ pub struct Field {
 pub struct Empty;
 
 /// Describes the fields present on a span.
```

Comment:
let's add a description to this comment explaining the equality rules (specifically, that two `FieldSet`s are equal only if they originate from the same callsite)